### ISSUE TITLE
Create an Opening Times component

### ISSUE THREAD

**Issue by IncognitoJam** [campus-bites/campus-bites-vue#33]
The opening times data in Firestore has the following structure:
```json
"openingTimes": {
  "monday": [{ "opens": "09:00", "closes": "04:00" }],
  ...
  "saturday": [{ "opens": "09:00", "closes": "04:00" }],
  "sunday": [{ "opens": "09:00", "closes": "01:45"} ]
},
```

This is the same as the data structure required by [moment-opening-times](https://github.com/nhsuk/moment-opening-times/), a package we can use to determine on the client whether or not a restaurant is open.

You should create a new Vue component which takes the opening times for a restaurant as a [Prop](https://vuejs.org/v2/guide/components-props.html) and displays in text the current status of the restaurant ("Open Now", "Closed Now").

**Future goals:**
 - Make the restaurant opening and closing times accessible somehow, for example with a dropdown. This should be configurable.
 - Display "Closing Soon" or "Opening Soon" when the restaurant is about to close or open respectively. This should be configurable.
 - Any other feature you think would be useful

**Comment by IncognitoJam** [comment]
Implemented in 6b27f17528c6237938990f6dcedf67c9faef2ef4. Needs review.

**Comment by IncognitoJam** [comment]
This was implemented in 6b27f17528c6237938990f6dcedf67c9faef2ef4, 144793d34ddaf6c72e5ecc661d9421da9eb35241 and 58448dd35ba4aec865387cc0a53bd14787dd5351.
